//Код програми 8.1. Демонстрація механізму оброблення винятків

#include <iostream>		 		// Для потокового введення-виведення
#include <cstdlib>				// Стандартна бібліотека С++

using namespace std; 			// Використання стандартного простору імен

int main()
{
    cout << "Початок" << endl;
    try { // Початок try-блоку
        cout << "У try-блоцi" << endl;
        throw 99; // Генерування помилки
        cout << "Ця настанова не буде виконана.";
    }
    catch(int c) { // Перехоплення помилки
        cout << "Значення перехопленого винятку дорiвнює: " << c << endl;
    }

    cout << "Кiнець програми";

    //system("PAUSE");
    return EXIT_SUCCESS;
}
